examples/js/loaders/STLLoader.js

```diff
@@ -245,22 +279,13 @@ THREE.STLLoader.prototype = {
 		if ( typeof buf !== "string" ) {
 
 			var array_buffer = new Uint8Array( buf );
+			var strArray = [];
+			for ( var i = 0; i < buf.byteLength; i ++ ) {
 
-			if ( window.TextDecoder !== undefined ) {
```

Comment:
sorry.. worked with a little previous version.. reverted this and commited only the fix. sorry 'bout that